src/components/NewDraftWizard/WizardForms/WizardDOIForm.js

```diff
@@ -0,0 +1,99 @@
+//@flow
+
+import React, { useEffect, useState } from "react"
+
+import { makeStyles } from "@material-ui/core/styles"
+import Ajv from "ajv"
+import { useForm, FormProvider } from "react-hook-form"
+import { useSelector, useDispatch } from "react-redux"
+
+import { WizardAjvResolver } from "./WizardAjvResolver"
+import JSONSchemaParser from "./WizardJSONSchemaParser"
+
+import { ResponseStatus } from "constants/responseStatus"
+import { resetAutocompleteField } from "features/autocompleteSlice"
+import { setOpenedDoiForm } from "features/openedDoiFormSlice"
+import { updateStatus } from "features/statusMessageSlice"
+import { resetCurrentObject } from "features/wizardCurrentObjectSlice"
+import { addDoiInfoToFolder } from "features/wizardSubmissionFolderSlice"
+import schemaAPIService from "services/schemaAPI"
+import { dereferenceSchema } from "utils/JSONSchemaUtils"
+
+const useStyles = makeStyles(theme => ({
+  form: theme.form,
+}))
+
+const DOIForm = ({ formId }: { formId: string }): React$Element<typeof FormProvider> => {
+  const [dataciteSchema, setDataciteSchema] = useState({})
+
+  useEffect(() => {
+    const getDataciteSchema = async () => {
+      let dataciteSchema = sessionStorage.getItem(`cached_datacite_schema`)
+      if (!dataciteSchema || !new Ajv().validateSchema(JSON.parse(dataciteSchema))) {
+        try {
+          const response = await schemaAPIService.getSchemaByObjectType("datacite")
+          dataciteSchema = response.data
+          sessionStorage.setItem(`cached_datacite_schema`, JSON.stringify(dataciteSchema))
+        } catch (err) {
+          dispatch(
+            updateStatus({
+              status: ResponseStatus.error,
+              response: err,
+              errorPrefix: "Can't submit the DOI information",
```

Comment:
`errorPrefix` has been named to `helperText` in `statusMessageHandler` and therefore `errorPrefix` won't render message prefix. This will be typed in near future and will cause flow error. 

```suggestion
              helperText: "Can't submit the DOI information",
```